### PR TITLE
CotE - In Defense of Rokugan

### PR DESCRIPTION
**Pull request by jamesoddy**
For issue #2605

### CODE REVIEW

File: server/game/cards/06-CotE/InDefenseOfRokugan.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            condition: () => this.game.isDuringConflict(),
+            cost: ability.costs.sacrifice(card => card.type === CardTypes.Character && card.isDefending()),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.isAttacking(),
+                gameAction: ability.actions.cardLastingEffect({
+                    effect: ability.effects.setMilitarySkill(0)
+                })
+            },
+            effect: 'effect'
```
Comment (jeremylarner) [inline server/game/cards/06-CotE/InDefenseOfRokugan.js]: You need a description of the effect here

File: server/game/cards/06-CotE/InDefenseOfRokugan.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            title: 'Set an attacking character to 0 military skill',
+            condition: () => this.game.isDuringConflict(),
+            cost: ability.costs.sacrifice(card => card.type === CardTypes.Character && card.isDefending()),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.isAttacking(),
+                gameAction: ability.actions.cardLastingEffect({
+                    effect: ability.effects.setMilitarySkill(0)
+                })
+            }
```
Comment (jeremylarner) [inline server/game/cards/06-CotE/InDefenseOfRokugan.js]: You still need an 'effect' here - the default for cardLastingEffect is 'X plays Y to apply a lasting effect to Z' which isn't particularly helpful

Comment (jamesoddy) [reply server/game/cards/06-CotE/InDefenseOfRokugan.js]: Right I get it now.  This is a bit of a magical string that you just have to know what it does.  I've just pushed a change.

File: server/game/cards/06-CotE/InDefenseOfRokugan.js [hunk 3]
```diff
@@ -14,7 +14,8 @@ class InDefenseOfRokugan extends DrawCard {
                     effect: ability.effects.setMilitarySkill(0)
                 })
             },
-            effect: 'set {0} to 0 military skill'
+            effect: 'set {0}\'s {1} skill to 0',
+            effectArgs: () => 'military'
```
[the remark this answers is not in the document]
Comment (jeremylarner) [reply server/game/cards/06-CotE/InDefenseOfRokugan.js]: yes, it formats the message to use the military icon

**Pull request merged**